contributing/checklist.md

```diff
@@ -62,17 +62,19 @@ The included catalog application uses Core Graphics to draw landing page tiles f
 
 #### To convert a raw asset to Core Graphics:
 
-1. Set the canvass size to 188 x 155.
+1. Set the canvass size to 82 x 82.
```

Comment:
nit: s/canvass/canvas/ 
(Since we are changing the line anyway.)

Reply:
Done